dev/reference/tools/tidb-lightning/config.md

```diff
@@ -0,0 +1,324 @@
+# TiKV cluster is performed at the end of the import.
+# The default value is `false`.
+compact = false
+# Performs `ANALYZE TABLE <table>` for each table.
+analyze = true
+
+# Configures the background periodic actions.
+# Supported units: h (hour), m (minute), s (second).
+[cron]
+# Duration between which Lightning automatically refreshes the import mode
+# status. Should be shorter than the corresponding TiKV setting.
+switch-mode = "5m"
+# Duration between which an import progress is printed to the log.
+log-progress = "5m"
+
+# Table filter options. See the corresponding section for details.
+#[black-white-list]
+# ...
+```
+
+### TiKV Importer
+
+```toml
+# TiKV Importer configuration file template.
+
+# Log file.
+log-file = "tikv-importer.log"
+# Log level: trace, debug, info, warn, error, off.
+log-level = "info"
+
+[server]
+# The listening address of tikv-importer. tidb-lightning needs to connect to
+# this address to write data.
+addr = "0.0.0.0:8287"
+# Size of the thread pool for the gRPC server.
+grpc-concurrency = 16
+
+[metric]
+# The Prometheus client push job name.
+job = "tikv-importer"
+# The Prometheus client push interval.
+interval = "15s"
+# The Prometheus Pushgateway address.
+address = ""
+
+[rocksdb]
+# The maximum number of concurrent background jobs.
+max-background-jobs = 32
+
+[rocksdb.defaultcf]
+# Amount of data to build up in memory before flushing data to the disk.
+write-buffer-size = "1GB"
+# The maximum number of write buffers that are built up in memory.
+max-write-buffer-number = 8
+
+# The compression algorithms used in different levels.
+# The algorithm at level-0 is used to compress KV data.
+# The algorithm at level-6 is used to compress SST files.
+# The algorithms at level-1 to level-5 are unused for now.
+compression-per-level = ["lz4", "no", "no", "no", "no", "no", "lz4"]
+
+[rocksdb.writecf]
+# (same as above)
+compression-per-level = ["lz4", "no", "no", "no", "no", "no", "lz4"]
+
+[import]
+# The directory to store engine files.
+import-dir = "/mnt/ssd/data.import/"
+# Number of threads to handle RPC requests.
+num-threads = 16
+# Number of concurrent import jobs.
+num-import-jobs = 24
+# Maximum duration to prepare Regions.
+#max-prepare-duration = "5m"
+# Split Regions into this size according to the importing data.
+#region-split-size = "512MB"
+# Stream channel window size. The stream will be blocked on channel full.
+#stream-channel-window = 128
+# Maximum number of open engines.
+max-open-engines = 8
+# Maximum upload speed (bytes per second) from Importer to TiKV.
+# upload-speed-limit = "512MB"
+# Minimum ratio of available space on the target store: `store_available_space`/`store_capacity`.
+# Importer pauses uploading SST if the availability ratio of the target store is less than this
+# value, to allow enough time for PD to balance Regions.
+min-available-ratio = 0.05
+```
+
+## Command line parameters
+
+### Usage of `tidb-lightning`
+
+| Parameter | Explanation | Overrides setting |
```

Comment:
Does `Overrides setting` mean that the setting in the 3rd column overrides that in the 1st column?

Reply:
Renamed the header to "Corresponding setting" and added an explanation.